docs/content/modeling/migrating/migrating-schema-1-1.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 sidebar_position: 2
 slug: /modeling/migrating/migrating-schema-1-1
+description: Migrate existing models to schema 1.1
```

Comment:
```suggestion
description: Migrate existing OpenFGA authorization models to schema version 1.1
```

Reply:
I will probably use `Migrate existing authorization models to schema version 1.1` instead